### PR TITLE
use a new method to download csv and vcf export file

### PR DESCRIPTION
**Pull request by RemiDesgrange**
This new method create a temporary button into the DOM in order to make
it feel like a real standard download from the browser.

### CODE REVIEW

**Comment by RemiDesgrange** [comment]
I think you can squash before merging.

File: console/src/main/webapp/manager/app/services/users.es6 [hunk 1]
```diff
@@ -54,7 +54,7 @@ angular.module('manager')
       })
     }
   }
-  ]).factory('ExportVCard', ['$http', 'CONSOLE_PRIVATE_PATH', ($http, baseUri) => {
+  ]).factory('ExportVCF', ['$http', 'CONSOLE_PRIVATE_PATH', ($http, baseUri) => {
```
Comment (fvanderbiest) [inline console/src/main/webapp/manager/app/services/users.es6]: seriously ?

File: console/src/main/webapp/manager/app/components/users/users.es6 [hunk 2]
```diff
@@ -81,32 +81,38 @@ class UsersController {
...
-    const download = this.$injector.get('ExportCSV')
+  export_ (fileType) {
+    const download = this.$injector.get(`Export${fileType.toUpperCase()}`)
     download(this.selection).then(result => {
       if (result.status !== 200) {
         throw new Error(`Cannot fetch users list. error ${result.status}`)
       }
-      const blob = new Blob([result.data], { type: 'text/csv' })
-      window.open(window.URL.createObjectURL(blob))
+      let extension = fileType
```
Comment (fvanderbiest) [inline console/src/main/webapp/manager/app/components/users/users.es6]: Seems overcomplicated ...

File: console/src/main/webapp/manager/app/components/users/users.es6 [hunk 3]
```diff
@@ -81,32 +81,40 @@ class UsersController {
...
+          mimetype = 'text/x-vcard'
+          break
+        default:
+          mimetype = `text/${fileType}`
+      }
+      const blob = new Blob([result.data], { type: mimetype })
+      const a = document.createElement('a')
+      a.href = window.URL.createObjectURL(blob)
+      a.target = '_blank'
+      a.download = `users.${fileType}`
```
Comment (fvanderbiest) [inline console/src/main/webapp/manager/app/components/users/users.es6]: Could you generate a filename with a timestamp like `20190723-100356_users_export.[csv|vcf]` ?

**Pull request merged**